### ISSUE TITLE
Transitioning to .NET 9

### ISSUE THREAD

**Issue by NinjataWRLD** [JasperFx/wolverine#1131]
**Describe the bug**
When I transitioned to .NET 9, warnings about vulnerabilities across various packages forced me to update them all to their latest version. This on the other hand caused a incompatibility error between Wolverine and the new packages - namely, packages like Microsoft.Extensions.Caching.Memory and Microsoft.Extensions.Configuration.Builder, of which you don't yet support the 9.0.0 version.

**To Reproduce**
1. Transition existing projects with dependency to WolverineFx to .NET 9
2. Install/Update packages with a transitive dependency to Microsoft.Extensions.Caching.Memory or Microsoft.Extensions.Configuration.Builder (e.g. Microsoft.EntityFrameworkCore, Microsoft.AspNetCore.Identity...) with the latest version, 9.0.0
3. Rebuild and go to Error List tab
4. See errors and warnings regarding the package mismatches

**Screenshots**
![image](https://github.com/user-attachments/assets/ca69f521-8d23-476d-a575-7c442fb81b2b)

**Desktop (please complete the following information):**
 - OS: Windows 11
 - IDE: Visual Studio Community 2022
 - Version: 17.12.0

**Comment by jeremydmiller** [comment]
Hey everybody, be more patient about the .NET 9 thing. I'm about to release it now, but just understand it's not magically possible for a tool like Wolverine to work with a new .NET version on day one. It's taken me about 2/3rds of a day to get the full critter stack on .NET 9, and that's after a couple other people helped as well doing prep work.

**Comment by NinjataWRLD** [comment]
I'm glad everything is fine now, sorry if we rushed you, and I admire your work - Wolverine is the perfect pakcage for me to use, I was so bummed out when I noticed it wasn't compatible with .NET 9 that I didn't even want to consider switching to other libraries😄.

Now, to the point... I updated to Wolverine 3.3, and the error and warnings were all resolved, but a new warning popped up. It's because I had the Microsoft.EntityFrameworkCore.Design package installed, which uses the Microsoft.CodeAnalysis.Common and Microsoft.CodeAnalysis.Workspaces.Common packages, both with v4.8.0.
Wolverine 3.3 uses both of those packages (JasperFx.CodeGeneration.Commands -> JasperFx.RuntimeCompiler -> Microsoft.CodeAnalysis 4.11, inside of which are nested the aforementioned packages).

I suppose the problem is in EF Core, since they use an older version and all? I'm not that good with NuGet stuff, so maybe it's really just my personal issue that can easily be resolved, but I have no idea how. I'm sorry for comment here again, but I want to see if other people are having this problem.

Anyway, I can build, compile and run my project just fine, and Wolverine works perfectly (happy to be able to use it again), and EF Core migration commands work as expected too - so I suppose I can neglect the two warnings until the EF Core updates their libraries. I've published an issue there too, I hope I'm not bothering people with problems they have nothing to do with though😥

**Comment by daniel-rutten-businessfundamentals-nl** [comment]
In case anyone else is struggling to get Wolverine up and running in .NET 9 with EF Core:

When I tried to install Wolverine, I got there errors:

NU1608: Warning As Error: Detected package version outside of dependency constraint: Microsoft.CodeAnalysis.Workspaces.MSBuild 4.8.0 requires Microsoft.CodeAnalysis.Common (= 4.8.0) but version Microsoft.CodeAnalysis.Common 4.11.0 was resolved.

NU1608: Warning As Error: Detected package version outside of dependency constraint: Microsoft.CodeAnalysis.Workspaces.MSBuild 4.8.0 requires Microsoft.CodeAnalysis.Workspaces.Common (= 4.8.0) but version Microsoft.CodeAnalysis.Workspaces.Common 4.11.0 was resolved.

So I uninstalled EF Core, upgraded the Microsoft.CodeAnalysis.Workspaces.MSBuild package to 4.8.11, reinstalled EF Core and Wolverine and all is hunky dory!
